trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -118,12 +118,15 @@ static final byte FLAG_TRACED = 1; // 00000001
 boolean traced = (traceOptions & FLAG_TRACED) == FLAG_TRACED
 ```
 
-#### Traced Flag (00000001)
+#### Requested Flag (00000001)
 When set, the least significant bit recommends the request should be traced. A caller who
 defers a tracing decision leaves this flag unset.
 
+#### Recorded Flag (00000010)
+When set, the least significant bit documents that the caller may have recorded trace data. A caller who does not record trace data out-of-band leaves this flag unset.
```

Comment:
Should we be more clear about the "Definitively not recorded" and "Maybe recorded" states here? Otherwise discussion will spur again, when an implementation cannot definitively declare if it recorded a trace ("undecided").